spdx/document.py

```diff
@@ -120,7 +120,6 @@ def __init__(
         self.version = version
         self.data_license = data_license
         self.name = name
-        self.license_list_version=license_list_version
```

Comment:
If we want backwards compatibility, we shouldn't erase fields from classes that people might be using.